### PR TITLE
[UI] Service Mesh updates

### PR DESCRIPTION
**Pull request by Chadha93**
Signed-off-by: Gaurav Chadha <gauravchadha1676@gmail.com>

**Description**
1. Added Cilium service mesh info.
2. Removed/shutdown(status) for Octarine service mesh.
3. Updated the service mesh timeline.
4. Removed typos.

This PR fixes #

**Notes for Reviewers**
Will create corresponding issues for meshery.io and docs.

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/collections/landscape/non-functional.js [hunk 1]
```diff
@@ -200,19 +199,6 @@ export const non_functional=[{
   timeline_order: 14,
   icon: SofaMesh
 },
-{
-  name: "Octarine",
-  opensource: "No",
-  link: "https://www.octarinesec.com/",
```
Comment (leecalcote) [inline src/collections/landscape/non-functional.js:206]: This non-functional is only part of the landscape update that would be needed (functional update needed, too).

Let's leave Octarine in the landscape, but list as Project Shutdown in the first column of the Functional table.

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: src/collections/landscape/non-functional.js (hunk 1 above)
Comment (Chadha93) [reply src/collections/landscape/non-functional.js]: Done

**Comment by l5io** [comment]
🚀 Preview for commit dec700ca6ff0fe618dcc58e793a031f33ebe83c9 at: https://6258640e238cc71034f2d9c7--layer5ng.netlify.app

**Comment by subhradwip01** [comment]
LGTM👌

**Comment by Nikhil-Ladha** [comment]
@leecalcote good to merge?

**Review by warunicorn19 — APPROVED** [review]
LGTM!

File: src/collections/landscape/meshes.js [hunk 2]
```diff
@@ -348,14 +348,15 @@ export const meshes=[
...
     desc: "Octarine is a security platform for k8s workloads that combines admission control with runtime network security leveraging an Envoy-based service mesh",
     link: "https://www.octarinesec.com/",
-    h2: "Yes",
-    grpc: "Yes",
-    tcp_web: "Yes",
-    multi_cluster: "Yes",
-    multi_tenant: "Yes",
-    prometheus: "No",
-    tracing: "No",
-    encryption: "Yes",
```
Comment (leecalcote) [inline src/collections/landscape/meshes.js:358]: @Chadha93 I actually wanted this left in.

File: src/collections/landscape/tools-data.js [hunk 3]
```diff
@@ -19,7 +19,7 @@ export const tools_data= [
     tool: "Meshery",
     link: "https://github.com/layer5io/meshery",
     language: "Golang",
-    written_for: "Istio, Linkerd, Consul, Octarine, Network Service Mesh, App Mesh",
```
Comment (leecalcote) [inline src/collections/landscape/tools-data.js:22]: good

**Pull request merged**